charts/app/templates/backend-sites/templates/deployment.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.backendSites.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "backendSites.fullname" . }}
+  labels:
+    {{- include "backendSites.labels" . | nindent 4 }}
+spec:
+  strategy:
+    type: {{ .Values.backendSites.deploymentStrategy }}
+  {{- if not .Values.backendSites.autoscaling.enabled }}
+  replicas: {{ .Values.backendSites.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "backendSites.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
+      labels:
+        {{- include "backendSites.labels" . | nindent 8 }}
+    spec:
+      {{- if .Values.backendSites.podSecurityContext }}
+      securityContext:
+        {{- toYaml .Values.backendSites.podSecurityContext | nindent 12 }}
+      {{- end }}
+      initContainers: # The container that will do the initial database seed and migrations.
```

Comment:
Great to see initContainer stuff here.

I'll note that this one IS using `sites-migration` as the image, whereas I think the earlier review comment spotted one just using `sites`. I assume these should be made the same?

Reply:
yeah the shcemaspy needs to be updated, this one is correct.